.github/workflows/run-tests.yml

```diff
@@ -75,3 +75,7 @@ jobs:
         uses: ammaraskar/sphinx-action@master
         with:
           docs-folder: "docs/"
+      - name: Check that docs are up to date
+        run: |
+          docker build --file=Dockerfile.tests --tag tests . && \
+          docker run tests
```

Comment:
We now have two main Dockerfile, the one that we use to run tests and the one that we use to build documentation. Why is the documentation one named `Dockerfile.tests`?

Reply:
Currently it does not build docs. It only checks (or tests) if the documentation templates are up to date.

Is there any other name you'd prefer? Let me move this file to `ci/Dockerfile` if it will make it clear that it is not a main Dockerfile.

Reply:
Well I think I would prefer `ci/Dockerfile.docs` (the move to ci/ is nice). And if the contents have to contain a copy/paste version of the main Dockerfile, can we script that, use a template/include mechanism (only in the ci/ version, leave the main Dockerfile alone/unchanged)?

Reply:
Moved the file to `ci/Dockerfile.docs.template`. One can run the following command to generate the final Dockerfile that is ignored in the git worktree:

```sh
cat Dockerfile ci/Dockerfile.docs.template > ci/Dockerfile.docs
```

I do not expect users to actually use this dockerfile in any way. I think `make update-docs` should be the command to run on development environments.